.github/workflows/linux-32bit-build-and-test-ignored.yaml

```diff
@@ -10,11 +10,13 @@ name: Regression Linux i386
       - '**.md'
       - 'LICENSE*'
       - NOTICE
+      - '.github/workflows/linux-32bit-build-and-test*.yaml'
```

Comment:
Maybe we can include all .yaml files in this directory?

Reply:
You're correct. Added all and also added it into 32bits and windows tests as well